packages/runtime/container-runtime/src/containerHandleContext.ts

```diff
@@ -13,7 +13,6 @@ import { generateHandleContextPath } from "@fluidframework/runtime-utils";
 import { ContainerRuntime } from "./containerRuntime";
 
 export class ContainerFluidHandleContext implements IFluidHandleContext {
-    public get IFluidRouter() { return this; }
```

Comment:
Kind of raises an interesting question of whether we have any other places where we falsely identify as an `IFluid*` without actually implementing the interface.

Reply:
it's probably happens. we should probably enforce a lint rule that force typing on properties, as that would give us a compile time failure.